schemas/2017-07-01-preview/cloudsimple.privatecloudiaas.json

```diff
@@ -0,0 +1,375 @@
+{
+  "id": "https://schema.management.azure.com/schemas/2017-07-01-preview/cloudsimple.privatecloudiaas.json#",
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "cloudsimple.privatecloudiaas",
+  "description": "cloudsimple privatecloudiaas Resource Types",
+  "resourceDefinitions": {
+    "virtualmachines": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string"
+        },
+        "type": {
+          "type": "string",
+          "enum": [
+            "cloudsimple.privatecloudiaas/virtualmachines"
```

Comment:
>cloudsimple.privatecloudiaas/virtualmachines [](start = 13, length = 44)

are these generated? Shouldn't this follow the casing conventions to be "CloudSimple.PrivateCloudIaas/virtualMachines"?

Reply:
yes, this is generated file but i changed name to correct one